WebApplication/ClientApp/src/App.js

```diff
@@ -37,13 +37,16 @@ export class App extends Component {
   componentDidMount() {
     this.props.fetchShowParametersChanged();
 
-    const params = window.location.search.substring(1);
+    const rawParams = window.location.search.substring(1);
```

Comment:
It would be good to have a comment here on what is going on, because now it's unclear. (and a sample of incoming data, which gets transformed)

Reply:
Will do :) 👍